core/src/main/java/org/mskcc/cbio/portal/servlet/QueryBuilder.java

```diff
@@ -243,79 +244,130 @@ protected void doPost(HttpServletRequest httpServletRequest,
             
             //this is a common case for most of the requests from study view when the there is only one study
             if (cancerStudyId != null && cancerStudyId.equals("all") && cancerStudyList != null && cancerStudyList.split(",").length == 1) {
-            		cancerStudyId = cancerStudyList.split(",")[0];
-            		addProfiles = true;
+                cancerStudyId = cancerStudyList.split(",")[0];
+                addProfiles = true;
             }
 
             //redirect requests with cancerStudyId="all" and cancerStudyList="all" to home page
             // handle different possible scenarios in https://github.com/cBioPortal/cbioportal/issues/3431
-            if (!(cancerStudyId != null 
```

Comment:
Nothing changed in this blob? Only formating?

Reply:
Yes!